src/course-outline/page-alerts/PageAlerts.test.jsx

```diff
@@ -103,9 +108,11 @@ describe('<PageAlerts />', () => {
     const discussionAlertDismissKey = `discussionAlertDismissed-${pageAlertsData.courseId}`;
     expect(localStorage.getItem(discussionAlertDismissKey)).toBe('true');
 
-    const feedbackLink = queryByText(messages.discussionNotificationFeedback.defaultMessage);
-    expect(feedbackLink).toBeInTheDocument();
-    expect(feedbackLink).toHaveAttribute('href', 'some-feedback-url');
+    await waitFor(() => {
+      const feedbackLink = getByText(messages.discussionNotificationFeedback.defaultMessage);
+      expect(feedbackLink);
+      expect(feedbackLink).toHaveAttribute('href', 'some-feedback-url');
+    });
```

Comment:
This was a flaky test

Reply:
Nit: you could just write `const feedbackLink = await findByText(...` instead of using `await waitFor(() => {`